topic_table_model.h

```diff
@@ -184,8 +184,13 @@ class TopicTableModel : public QAbstractTableModel
 
     bool check_rc(DDS::ReturnCode_t rc, const char* what);
 
+    void setDataRow(DataRow* const row, const DDS::DynamicData_var& data, DDS::MemberId id);
+
+    void parseCollection(const DDS::DynamicData_var& data, std::string namePrefix);
```

Comment:
```suggestion
    void parseCollection(const DDS::DynamicData_var& data, const std::string& namePrefix);
```

Not absolutely necessary, but we typically use pass by const reference for these (also the two functions below)